pyanaconda/ui/gui/spokes/subscription.py

```diff
@@ -840,6 +843,16 @@ def _register(self):
         # wait for the previous subscription thread to finish
         threadMgr.wait(THREAD_SUBSCRIPTION)
 
+        # check if the current installation source will be overriden
+        # and remember it if it is the case
+        source_proxy = self.payload.get_source_proxy()
+        source_type = source_proxy.Type
+        if source_type in SOURCE_TYPES_OVERRIDEN_BY_CDN:
+            self._overridden_source_type = source_type
```

Comment:
Please, keep `self.payload.Sources`. There is no reason to make it so complicated.